spec/order_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../lib/order'
+
+describe Order do
+
+  it 'shows the menu properly' do
+    hash_example = { dish: :price }
+    subject = Order.new hash_example, 'dummy_twilio'
+    expect(subject.show_menu).to eq("#{:dish}, £ #{:price}\n")
```

Comment:
Beware testing presentation strings quite so precisely as they can easily change.  You might instead test that the name is included in the string and separately that the price is included.